tests/kafkatest/services/kafka/kafka.py

```diff
@@ -926,7 +932,7 @@ def run_features_command(self, op, new_version):
     def pids(self, node):
         """Return process ids associated with running processes on the given node."""
         try:
-            cmd = "jcmd | grep -e %s | awk '{print $1}'" % self.java_class_name()
```

Comment:
`jcmd` command won't work if Kafka is running in the native mode.
Replacing it with `ps ax`.